iso.bbx

```diff
@@ -315,18 +400,33 @@
     {\printfield{howpublished}}%
 }
 
-%publication info
+% PUBLICATION INFO MACROS
+
+% By default only year is used, but for some entries,
+% we need full date (day or even time precision),
+% (e.g. patents, TV shows, newspapers, online news)
+
+% Hack to print full date (follow ISO8601 format YYYY-MM-DD)
```

Comment:
I really don't like this hack. I appreciate that until we get something done for https://github.com/plk/biblatex/issues/863 some clever workarounds are needed, but the previous trick seemed much cleverer to me (I haven't checked if the previous hack still works, but it should be possibly to get something similar going again if t stopped working).

Reply:
Me neither, but I didn't know how to print the full date, because the previous code
```
\printtext{\csname mkbibrangeiso8601\endcsname{\thefield{date}}}%
```
didn't work anymore and I came up only with this hack. Now I committed 964eff1 that sticks to the old trick and seems to work, but I have no further insight into this..